econml/ortho_forest.py

```diff
@@ -855,3 +908,113 @@ def _check_treatment(self, T):
         except Exception as exc:
             raise ValueError("Expected numeric array but got non-numeric types.")
         return T
+
+
+class BLBInference(Inference):
+    """
+    Bootstrap-of-Little-Bags inference implementation for the OrthoForest classes.
+
+    This class can only be used for inference with any estimator derived from :class:`BaseOrthoForest`.
+
+    Parameters
+    ----------
+    estimator : :class:`BaseOrthoForest`
+        Estimator to perform inference on. Must be a child class of :class:`BaseOrthoForest`.
+    """
+
+    def fit(self, estimator, *args, **kwargs):
+        """
+        Fits the inference model.
+
+        This is called after the estimator's fit.
+        """
+        self._estimator = estimator
+        # Test expansion of treatment
+        # If expanded treatments are a vector, flatten const_marginal_effect_interval
+        _, T0, _ = self._estimator._expand_treatments(None, 0, 1)
+        self._T_vec = False
```

Comment:
I don't think you need the `if` below, this whole block could just be

```suggestion
        self._T_vec = T0.ndim == 1
```

Reply:
Fixed!